tests/CssToInlineStylesTest.php

```diff
@@ -60,7 +61,7 @@ public function testInlineStylesBlock()
         $this->cssToInlineStyles->setUseInlineStylesBlock();
         $this->cssToInlineStyles->setHTML($html);
         $actual = $this->findAndSaveNode($this->cssToInlineStyles->convert(), '//a');
-        $this->assertEquals($expected, $actual);
+        self::assertEquals($expected, $actual);
```

Comment:
... but in general it's bad practice to use static method with "this", maybe I need to re-configure my IDE-settings to ignore some warnings in test-files :) thx for the feedback!